ep_testing/config.py

```diff
@@ -41,8 +44,10 @@ def __init__(self, run_config_key, msvc_version=None):
         self.msvc_version = None
         if msvc_version is not None:
             self.msvc_version = msvc_version
+        elif self.os == OS.Windows and self.os_version == '2022':
+            self.msvc_version = 17
         elif self.os == OS.Windows:
-            self.msvc_version = 16  # Default to 2017 for Travis
+            self.msvc_version = 16
```

Comment:
If we are on Windows && 2022, use Visual Studio 17 (2022), else if we are on Windows (anything else), use Visual Studio 16 (2019).